Jenkinsfile

```diff
@@ -16,13 +27,32 @@ def deploy(config) {
   // Configure the environment, perform any database migrations if necessary,
   // start the rolling update, and finally, monitor until complete.
   dir('k8s') {
-    sh """
-      . config/${config}.sh
-      export APP_IMAGE_TAG=${tag}
-      make k8s-db-migration-job
-      make k8s-deployments
-      make k8s-rollout-status
-    """
+    try {
+
+      notify_slack([
+        status: "Pushing to ${environment}",
+        message: "developer-portal image ${tag}"
+      ])
+
+      sh """
+        . config/${config}.sh
+        export APP_IMAGE_TAG=${tag}
+        make k8s-db-migration-job
+        make k8s-deployments
+        make k8s-rollout-status
+      """
+      notify_slack([
+        stage: "Deployed to ${environment}",
+        status: 'sucess'
```

Comment:
```suggestion
        status: 'success'
```

Reply:
Fixed in f839bab874e44aea0e3f4d82e3123cf588b5ec04